docker-compose.yml

```diff
@@ -4,17 +4,33 @@ services:
     build:
       context: .
     container_name: "ambnode"
+    command: yarn dev:start
     environment:
-      - NODE_ENV=production
       - MONGODB_URI=mongodb://db:27017/ambrosus_node
       - MONGODB_DATABASE=ambrosus_node
-      - WEB3_RPC=http://167.99.138.196:8545
+      - WEB3_RPC=http://parity:8545
```

Comment:
You have removed support for the WEB3_RPC env variable, so this should also be removed.